src/components/state-history-chart-line.html

```diff
@@ -160,44 +166,7 @@
           prevValues = values;
         }
 
-        if (domain === 'thermostat') {
```

Comment:
Why remove this? This means that for thermostats we are no longer plotting target and current temperatures